flutter/shell/platform/tizen/tizen_window_elementary.cc

```diff
@@ -426,6 +431,35 @@ void TizenWindowElementary::PrepareInputMethod() {
       [this]() { view_delegate_->OnComposeEnd(); });
   input_method_context_->SetOnCommit(
       [this](std::string str) { view_delegate_->OnCommit(str); });
+
+#ifdef AUTOFILL_SUPPORT
+  ctxpopup_ = elm_ctxpopup_add(elm_win_);
+
+  input_method_context_->SetOnPopupAutofillContext([this]() {
+    if (TizenAutofill::GetInstance().GetAutofillItems().size() > 0) {
```

Comment:
If clear all items before starting to add items, the `EVAS_CALLBACK_HIDE` for `ctxpopup_` no longer needed??

```suggestion
  input_method_context_->SetOnPopupAutofillContext([this]() {
    elm_ctxpopup_clear(ctxpopup_);
    if (TizenAutofill::GetInstance().GetAutofillItems().size() > 0) {
```

Reply:
If change it as you said, we won't need `EVAS_CALLBACK_HIDE`, but the previous information will remain until `requestAutofill` is executed.

Would it be better to implement it so that it doesn't use `EVAS_CALLBACK_HIDE`?

Reply:
Um... Is there a path for the popup to be hidden other than when an item is selected? 
If so, I think it is good to clear the item through the `EVAS_CALLBACK_HIDE` callback.
also, when an item is selected, it would be nice to clear all items immediately after committing.

Reply:
When `ctxpopup` loses focus, `ctxpopup` is automatically hidden.